inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/sinks/SenderManager.java

```diff
@@ -62,6 +62,7 @@ public class SenderManager {
     private static final Logger LOGGER = LoggerFactory.getLogger(SenderManager.class);
     private static final SequentialID SEQUENTIAL_ID = SequentialID.getInstance();
     private static final AtomicInteger SENDER_INDEX = new AtomicInteger(0);
+    private static final String PROTOCOL_TYPE_TCP = "tcp";
```

Comment:
Suggest move the `org.apache.inlong.manager.common.consts.ProtocolType` to inlong-common(`org.apache.inlong.common.constant`), and use it in Agent, DataProxy, etc.